packages/main/src/Input.hbs

```diff
@@ -40,6 +40,7 @@
 
 		{{#if showSuggestions}}
 			<span id="{{_id}}-suggestionsText" class="ui5-hidden-text">{{suggestionsText}}</span>
+			<span id="{{_id}}-selectionText" class="ui5-hidden-text" aria-live="polite" role="status"></span>
```

Comment:
bind a new private metadata state property here,inside the span, f.e. {{_announcementText}}
don't forget to declare it in `metadata`